### DISCUSSION TITLE
ebesucher running with proxies but not earning

### DISCUSSION THREAD

**Discussion by peercoder** [engageub/InternetIncome#110] [General]
Hi,
I'm your referred 
I am earning great with ebesucher with proxies but recently ebesucher docker containers still running but not earning anymore
I don't know what is caused problems: proxies or code or ebesucher change?
Thanks your great code!

**Comment by copystring** [comment]
Same problem here. I'm sure it's ebesucher blocking proxies. Need to find good proxies working with ebesucher.

**Reply by engageub** [reply]
@copystring Could you please let me know if this is not working with custom firefox also in test branch?

**Reply by copystring** [reply]
Yes, it is not working in test branch with Firefox, but it does not matter. Chrome does not work either because of my proxies. I cannot find good and fast proxies that work with ebesucher and honeygain. Some work with ebesucher but don't work well with honeygain because they are slow. So I don't know what to do anymore.

**Reply by engageub** [reply]
CUSTOM_FIREFOX_USE_RANDOM_DISPLAY has been added to test branch now. You may enable this in properties.conf for custom firefox. It generates random display size for each container.

**Reply by copystring** [reply]
Thank you, but this will not solve my problem, I cannot find good proxies. I have some proxies, that work with Firefox in test branch, but they are so slow, they don't work with honeygain. CUSTOM_FIREFOX_USE_RANDOM_DISPLAY cannot change this :)

**Comment by engageub** [comment]
Hello,
If you are using main branch with firefox, there is an issue with captcha solver.
Recently, it has been updated to use chrome by default. 
Please download the script again and make sure you are using chrome.

Thank you

**Reply by peercoder** [reply]
I tried using chrome but not earning too and get this container logs: Obt-Message: Xinerama extension is not present on the server
I also changed proxies it earnings some hours then stops earning so i think ebesucher blocking proxies
Thanks your support!

**Reply by engageub** [reply]
Hello,
You have to access the GUI url which starts with http://127.0.0.0:... 
Ebesucher does not  usually support data center IPs. It shows 0 points even if the surfing starts.
Please access the GUI interface of the container to see what is going on in the browser. 
Please see [FAQ](https://github.com/engageub/InternetIncome?tab=readme-ov-file#-how-to-access-127001-urls-for-ebesucher-or-mystnodes-if-gui-is-not-available-in-linux) on how to access ebesucher URL. 

Thank you

**Reply by peercoder** [reply]
Thanks your guide, I accessed the GUI url, there a captcha on firefox, i switched to chrome to bypass captcha, surfing starts but not earnings it shows 0 BTP, so i known problem is my proxies, i need good residential proxies.
My proxies still work with pawns but not ebesucher. What proxies work good with ebesucher?
Thank you

**Comment by engageub** [comment]
> Thanks your guide, I accessed the GUI url, there a captcha on firefox, i switched to chrome to bypass captcha, surfing starts but not earnings it shows 0 BTP, so i known problem is my proxies, i need good residential proxies.

It depends on the location and ISP of the proxy provider. Proxies which work now, may not work later if their score is low. You have to test accordingly before purchasing.
